kotlin/integrated-windows-authentication/README.md

```diff
@@ -1,32 +1,47 @@
-# Integrated Windows Authentication
+# Integrated windows authentication
 
-Uses Windows credentials to access services hosted on a portal secured with Integrated Windows Authentication (IWA).
+Connect to an IWA secured Portal and search for maps.
 
-![Integrated windows authentication App](integrated-windows-authentication.png)
+![Image of integrated windows authentication](integrated-windows-authentication.png)
 
 ## Use case
 
-IWA, which is built into Microsoft Internet Information Server (IIS), works well for intranet applications, but isn't always practical for internet apps.
+Your organization might use Integrated Windows Authentication (IWA) to secure ArcGIS Enterprise. This can be useful because the same credentials used to log into your work computer and network can be used to authenticate with ArcGIS. IWA is built into Microsoft Internet Information Server (IIS) and works well for intranet applications but isn't always practical for internet apps.
+
 
 ## How to use the sample
 
```

Comment:
I believe this section is not supposed to use the 1.2.3. notation and instead should be a paragraph. You could probably just remove the newlines and numbers